charts/acp/values.yaml

```diff
@@ -148,12 +148,23 @@ certManager:
 
 importJob:
   enabled: false
+  args:
```

Comment:
I would rather not force users to overwrite the args completely if they want to change sth. Can we have sth like this:
```yaml
importJob:
  enabled: false
  # path to config files
  config: ["/data/config,yaml", "/secret/config.yaml"]
  # import mode (update, fail, ignore)
  mode: update
  # input file format: yaml or json
  format: yaml
  # path to the input file
  input: /import/seed.yaml
  # extra args for import command
  extraArgs: []
```
in values.yaml instead and build the args list in the template?

Reply:
Updated